Detectors/PHOS/calib/include/PHOSCalib/CalibParams.h

```diff
@@ -0,0 +1,117 @@
+  void setGain(unsigned short cellID, float c) { mGainCalib[cellID] = c; }
+
+  /// \brief Set High Gain energy calibration coefficients for one module in the form of 2D histogram
+  /// \param 2D(64,56) histogram with calibration coefficients
+  /// \param module number
+  /// \return Is successful
+  bool setGain(TH2* h, int module);
+
+  /// \brief Get High Gain to Low Gain ratio calibration coefficients
+  /// \param cellID Absolute ID of cell
+  /// \return High Gain to Low Gain ratio of the cell
+  float getHGLGRatio(unsigned short cellID) { return mHGLGRatio.at(cellID); }
+
+  /// \brief Set High Gain to Low Gain ratio
+  /// \param cellID Absolute ID of cell
+  /// \param r is the calibration coefficient
+  void setHGLGRatio(unsigned short cellID, float r) { mHGLGRatio[cellID] = r; }
+
+  /// \brief Set High Gain to Low Gain ratio for one module in the form of 2D histogram
+  /// \param 2D(64,56) histogram with High Gain to Low Gain ratio
+  /// \param module number
+  /// \return Is successful
+  bool setHGLGRatio(TH2* h, int module);
+
+  /// \brief Get High Gain time calibration coefficients
+  /// \param cellID Absolute ID of cell
+  /// \return high gain time calibration coefficient of the cell
+  float getHGTimeCalib(unsigned short cellID) { return mHGTimeCalib.at(cellID); }
+
+  /// \brief Set High Gain time calibration coefficient
+  /// \param cellID Absolute ID of cell
+  /// \param t is the calibration coefficient
+  void setHGTimeCalib(unsigned short cellID, float t) { mHGTimeCalib[cellID] = t; }
+
+  /// \brief Set High Gain time calibration coefficients for one module in the form of 2D histogram
+  /// \param 2D(64,56) histogram with calibration coefficients
+  /// \param module number
+  /// \return Is successful
+  bool setHGTimeCalib(TH2* h, int module);
+
+  /// \brief Get Low Gain time calibration coefficient
+  /// \param cellID Absolute ID of cell
+  /// \return low gain time calibration coefficient of the cell
+  float getLGTimeCalib(unsigned short cellID) { return mLGTimeCalib.at(cellID); }
+
+  /// \brief Set time calibration coefficient
+  /// \param cellID Absolute ID of cell
+  /// \param t is the calibration coefficient
+  void setLGTimeCalib(unsigned short cellID, float t) { mLGTimeCalib[cellID] = t; }
+
+  /// \brief Set Low Gain time calibration coefficients for one module in the form of 2D histogram
+  /// \param 2D(64,56) histogram with calibration coefficients
+  /// \param module number
+  /// \return Is successful
+  bool setLGTimeCalib(TH2* h, int module);
+
+ private:
+  std::array<float, 14337> mGainCalib;   ///< Container for the gain calibration coefficients
+  std::array<float, 14337> mHGLGRatio;   ///< Container for the High Gain to Low Gain ratios
+  std::array<float, 14337> mHGTimeCalib; ///< Container for the High Gain time calibration coefficients
+  std::array<float, 14337> mLGTimeCalib; ///< Container for the Low Gain time calibration coefficients
```

Comment:
could you please define a constexpr constant for 14337 and use the constant in the code?